benchmarkapps/BasicApi/Controllers/StoreController.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Mvc;
+
+namespace BasicApi.Controllers
+{
+    [Route("/store")]
+    public class StoreController : ControllerBase
+    {
+        [HttpGet("inventory")]
+        public IActionResult GetInventory()
```

Comment:
Are these to drum up additional actions?

Reply:
I don't feel strongly about this, these could be here or be deleted. Your choice